src/unity/python/turicreate/test/test_image_similarity.py

```diff
@@ -238,9 +238,16 @@ def setUpClass(self):
                                                               input_image_shape=(3, 227, 227))
 
 
-@unittest.skipIf(_mac_ver() < (10,14), 'VisionFeaturePrint_Screen only supported on macOS 10.14+')
-class ImageSimilarityVisionFeaturePrintScreenTest(ImageSimilarityTest):
+@unittest.skipIf(_mac_ver() < (10,14), 'VisionFeaturePrint_Scene only supported on macOS 10.14+')
+class ImageSimilarityVisionFeaturePrintSceneTest(ImageSimilarityTest):
     @classmethod
     def setUpClass(self):
-        super(ImageSimilarityVisionFeaturePrintScreenTest, self).setUpClass(model='VisionFeaturePrint_Screen',
-                                                                            input_image_shape=(3, 299, 299))
+        super(ImageSimilarityVisionFeaturePrintSceneTest, self).setUpClass(model='VisionFeaturePrint_Scene',
```

Comment:
There is a syntax error at the end of this line. That's why the unit tests are failing.

FYI - I've refactored theses unit tests, so they should run pretty fast, if you want to run them locally.